### ISSUE TITLE
Multiple class id confused with VoTT and Detected result

### ISSUE THREAD

**Issue by lrscse** [AntonMu/TrainYourOwnYOLO#168]
Hi @AntonMu  i tried my custom data with single class that was very successful.. and thank you.. but on another custom data with 6 classes i am now confused on class id. With VoTT ( class start from 0 to 6) but how can i know which class is in detection result .csv file ( label start from 0 to 5 only). Because of this i guess i am not able to calculate mAP. 

Advance thanks

**Comment by AntonMu** [comment]
Hi @lrscse, 

The class names are the ones specified in [`classes.txt`](https://github.com/AntonMu/TrainYourOwnYOLO/blob/master/Data/Model_Weights/data_classes.txt) and are in that order. So the first entry has id 0, the second id 1, and so on. 

Hope this helps.

**Comment by lrscse** [comment]
Hi @AntonMu  thanks a lot sir. I am glad thak with your tutorial i am successful on my first work with single class. Now for second work its 6 classes, on training 300 images with 100 epoch on first stage ealry stoopin done in 124 epoch will loss and val_loss stuck at around 90.. because of this ( i guess) confidence score on detection is only arund 50-80 not much higher then this. how can i increase this?

**Comment by AntonMu** [comment]
First off, the detection confidence score is not the same as your performance of the detector. Because you now have 6 classes, it is expected that your detector has lower score for each class. With 6 classes, guessing randomly would just give you 16% confidence score - so 50% is pretty good. 

On the other hand, it also depends on your training images and their similarity. Try to have a look at the actual detection results (i.e. did it recognize the object I expected) and then perhaps label more images from classes it often gets confused. Overall, your training set size of 300 images for 6 classes is very small. It means that you only have 50 images per class. 

Hope this helps.

**Comment by lrscse** [comment]
@AntonMu  ok. i will see where it could be possible.. By the way.. its dental data so atleast i draw 15(minimum) ground truth per image i.e minimum 4-8 ground truth  of each class in an image

**Comment by AntonMu** [comment]
Ok - if you use dental data you also need to consider that your data is very different from what the entire network was trained on ([ImageNet](http://www.image-net.org/) - which is mostly animals, plants and food. Thus, transfer learning won't be as effective as it could be.

**Comment by lrscse** [comment]
Oh thanks for tips... anyway let me see.. it worked properly on my first worked which is also medical image
